src/Twig/Extension/CanonicalizeExtension.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Twig\Extension;
+
+use Symfony\Component\HttpFoundation\RequestStack;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class CanonicalizeExtension extends AbstractExtension
```

Comment:
The new classes contain only dead code that will never be executed in the current stable release. Either do this change on the next major branch or try to bridge this code in the old extensions

Reply:
The idea was to bridge the code, this is how it is looking like:

https://github.com/sonata-project/SonataAdminBundle/blob/5a1d5691a3334a8c4ee53318f58f6130ce5ca7b9/src/Twig/Extension/SonataAdminExtension.php#L632-L648